rosbridge_port: 9090
robot_namespaces:
  - ""
robot_ips:
  - ""
